src/modules/dummypythonqt/main.py

```diff
@@ -168,6 +173,8 @@ def retranslate(self, locale_name):
 # same rules apply as for ViewStep. No decorators are necessary here, because
 # only the ViewStep implementation is the unique entry point, and a module can
 # have any number of jobs.
+
+
```

Comment:
right - try the pep8 check and you will see that the check whining without two lines before a class definition :)